indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -41,20 +41,34 @@
     global.setupAbstractPage = function setupAbstractPage() {
         $('body').on('declarative:success', '.js-delete-comment', function() {
             $(this).closest('.i-timeline-item').remove();
+        }).on('ajaxForm:show', '.js-edit-comment', function() {
+            $(this).closest('.abstract-comment-badges').hide();
+        }).on('ajaxForm:hide', '.js-edit-comment', function() {
+            $(this).closest('.abstract-comment-badges').show();
         }).on('focus', '.new-comment textarea', function() {
-            var $commentForm = $(this).closest('form');
+            var $this = $(this);
+            var $box = $this.closest('#abstract-timeline-input');
+            var $commentForm = $this.closest('form');
```

Comment:
`$('#abstract-timeline-input .review-trigger')` is not so bad IMO - it just gets an item in a uniquely identified container